XBMC Remote/ConvenienceMacros.h

```diff
@@ -41,6 +41,7 @@
  * Other
  */
 #define LOCALIZED_STR(string) NSLocalizedString(string, nil)
+#define LOCALIZED_STR_ARGS(string, args...) [NSString stringWithFormat:LOCALIZED_STR(string), args]
```

Comment:
unfortunately, it's not as easy as that - it was only a pseudocode :) you'd need this:
```suggestion
#define LOCALIZED_STR_ARGS(string, ...) [NSString stringWithFormat:LOCALIZED_STR(string), __VA_ARGS__]
```